package.json

```diff
@@ -50,7 +50,7 @@
     "graphql": "^0.10.0 || ^0.11.0 || ^0.12.0 || ^0.13.0 || ^14.0.0"
   },
   "dependencies": {
-    "graphql-language-service-interface": "^1.3.2",
+    "graphql-language-service-interface": "^2.0.0",
```

Comment:
I think this is a semver-major change, because of the underlying change:

https://github.com/graphql/graphql-language-service/commit/895e68537fd802b8b6ddf2578a1f76f85982c773

This means we should also change our GraphQL versions to match. If we're going semver-major anyway, might be worth setting our base graphql level to 14.x (or 0.13.x if there's a compelling reason to do so).

Reply:
this is a great idea. I'll add this to this PR. Still awaiting credentials to release

Reply:
re: base graphql peer level, that's important to consider

Reply:
@benjie resolved you think? i just followed suit with gls above for now.

Reply:
Yeah; just make sure this goes out as 0.9.0 or higher 👍

Reply:
or maybe 14.x and same with GLS? looks like we have releases pending with minor version bumps for all, so once we get access to npm, we could just follow the `graphql-js` semver off the bat? i guess that's what I thought you were referring to at first but I see you were talking about the peer deps.

Reply:
That makes sense, we are tied deeply to graphql-js so using their major versioning makes sense (I don't think we need to match minor/patch though).

Reply:
@benjie i suppose we should do this from the monorepo?